src/main/java/http/server/App.java

```diff
@@ -7,9 +7,11 @@
 
 public class App {
     public static void main(String[] args) {
+        String portNumber = args.length > 0 ? args[0] : "5000";
+
         try {
             RequestRouter router = routerSetup();
-            Server server = new Server(new ServerSocket(5000), router);
+            Server server = new Server(new ServerSocket(Integer.parseInt(portNumber)), router);
```

Comment:
great, thanks! two more ideas:
- new ServerSocket can accept just portNumber integer, parseInt can happen one step prior
- "5000" can be some const like DEFAULT_PORT